cmd/gaia/app/app.go

```diff
@@ -170,7 +170,30 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 	)
 	app.SetInitChainer(app.initChainer)
 	app.SetBeginBlocker(app.BeginBlocker)
-	app.SetAnteHandler(auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper))
+
+	authAnteHandler := auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper)
+
+	filterAnteHandler := func(ctx sdk.Context, tx sdk.Tx, simulate bool) (newCtx sdk.Context, result sdk.Result, abort bool) {
+		newCtx, result, abort = authAnteHandler(ctx, tx, simulate)
+		if abort == true {
+			return
+		}
+
+		for _, msg := range tx.GetMsgs() {
+			switch msg.(type) {
+			case bank.MsgSend:
+				// LCD tests requires this to be commented out
+				// return ctx, bank.ErrSendDisabled(bank.DefaultCodespace).Result(), true
```

Comment:
Why? We should modify those tests, no?

Reply:
This doesn't seem safe, now we'll allow sends (?)

Reply:
Can I use the Makefile to set an environment variable or something that I can read from in the code?  Is that a thing?  Is that safe?

Reply:
we can't use an env variable but we can use a build flag, i'll do this.